src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -1149,6 +1154,22 @@ ExecuteRemoteCommand(const char *nodeName, uint32 nodePort, StringInfo queryStri
 Node *
 ParseTreeNode(const char *ddlCommand)
 {
+	Node *parseTreeNode = ParseTreeRawStmt(ddlCommand);
+
+#if (PG_VERSION_NUM >= 100000)
+	parseTreeNode = ((RawStmt *) parseTreeNode)->stmt;
+#endif
+
+	return parseTreeNode;
+}
+
+
+/*
+ * Parses the given DDL command, and returns the tree node for parsed command.
+ */
+Node *
+ParseTreeRawStmt(const char *ddlCommand)
+{
```

Comment:
Because using raw parsetrees is just about always wrong.  It's really hard to correctly work with them, because no semantic analysis has happened yet, and we can't do semantic analysis at that point, because referenced relations don't exist.  Because it happens inside of functions, we need separate infrastructure for CONCURRENTLY, VACUUM etc.